src/net.cpp

```diff
@@ -1100,10 +1100,16 @@ void ThreadDNSAddressSeed()
         } else {
             vector<CNetAddr> vIPs;
             vector<CAddress> vAdd;
-            if (LookupHost(seed.host.c_str(), vIPs))
+
+            //if the seed supports xored output, use a random subdomain x+(a-z)
+            //x[a-z].<NS> indicates that the dns seed server should response with XORed ips
+            if (LookupHost(((seed.fXoredIPs ? "x"+std::string(1, static_cast<char>(97+(int)GetRand(25)))+"." : "") + seed.host).c_str(), vIPs))
```

Comment:
To avoid collision. But probably no one will run a seed with [a-z].<domain.etc>.
IMO the x can be removed. On the other hand it is a more clear indicator of when a response should be xored.
So unsure.